### PR TITLE
Add --log-bytes, exposes transport.traceBytes qpid-jms option

### PR DESCRIPTION
**Pull request by jdanekrh**

### CODE REVIEW

File: cli-qpid-jms/src/test/kotlin/MainTest.kt [hunk 1]
```diff
@@ -122,32 +125,25 @@ class AacMainTest : AbstractMainTest() {
...
-//                "sender --log-msgs dict --broker amqp://127.0.0.1:5672 --address lalaLand --count 1".split(" ").toTypedArray()
-//        val receiverParameters =
-//                "receiver --log-msgs dict --broker amqpws://127.0.0.1:5672 --address lalaLand --count 1".split(" ").toTypedArray()
-////        print("Sending: ")
-////        Main.main(senderParameters)
-//        print("Receiving: ")
-//        Main.main(receiverParameters)
-//    }
-//
-//    @Test fun sendAndReceiveListMessage() {
```
Comment (enkeys) [inline cli-qpid-jms/src/test/kotlin/MainTest.kt]: Which is reason to deletion test for list?

File: cli-qpid-jms/src/main/resources/log4j.properties [hunk 2]
```diff
@@ -34,8 +34,9 @@ log4j.additivity.com.redhat.mqe.jms.Statistics=false
 
 log4j.logger.io.netty=WARN, console
 log4j.logger.netty=WARN, console
-# log4j.logger.org.apache.qpid=WARN, console
```
Comment (enkeys) [inline cli-qpid-jms/src/main/resources/log4j.properties]: Can be useful in some cases to have it in code.

File: cli-qpid-jms/src/test/kotlin/MainTest.kt (hunk 1 above)
Comment (jdanekrh) [reply cli-qpid-jms/src/test/kotlin/MainTest.kt]: It lives in https://github.com/rh-messaging/cli-java/blob/299a150b193f0b1d1aa4d2fe51dc9592b7931d05/tests/src/test/kotlin/AbstractMainTest.kt.

File: cli-qpid-jms/src/main/resources/log4j.properties (hunk 2 above)
Comment (jdanekrh) [reply cli-qpid-jms/src/main/resources/log4j.properties]: I will keep it, then. Maybe add comment, This is what you uncomment to get moar logz

Comment (jdanekrh) [reply cli-qpid-jms/src/main/resources/log4j.properties]: done

File: cli-qpid-jms/src/test/kotlin/MainTest.kt (hunk 1 above)
Comment (jdanekrh) [reply cli-qpid-jms/src/test/kotlin/MainTest.kt]: not a problem

**Pull request merged**